HW8.5/recursionFunctions.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#define _USE_MATH_DEFINES
+#include <math.h>
+#include "recursionFunctions.h"
+
+//functioons from task 2
+int factorialRecursion(int count)
+{
+	if (count == 0 || count == 1)
+	{
+		return 1;
+	}
+	return count * factorialRecursion(count - 1);
+}
+
+
+//functioons from task 3
+void printNaturalNumbersRecursionLessToHigh(int count)
+{
+	if (count > 9 || count < 0)
```

Comment:
what's up with 9? 
Count is the one to define number of elements, not const value